pytest.ini

```diff
@@ -14,7 +14,6 @@ markers =
     # Data collection markers
     skip_must_gather_collection: skip must gather collection on failures
     gpu: indicates collecting relevant namespace data on failure
-    swap: indicates collecting relevant namespace data on failure
```

Comment:
Are you sure you want to remove this?

Reply:
it's a duplicate
https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/pytest.ini#L30
we have same marker at lines 17 and 30 (so removing one of them)

Reply:
in fact same for `gpu` marker :) lines 16 and 32

Reply:
since you anyway touching it - can you please remove "gpu" as well